jekyll/_cci2/how-to-override-config.adoc

```diff
@@ -0,0 +1,344 @@
+
+== Understanding the override-with key
+The `override-with` key allows you to substitute a job in your workflow with a job defined in an orb.
+
+When you use the `override-with` key in a workflow, the following processes occur:
+
+* If the job referenced under `override-with` exists, the job definition from the referenced orb replaces your original job configuration.
+* If the referenced orb job does not exist, your original job definition is used as a fallback.
+
+See the xref:configuration-reference#override-with[`override-with` key reference] for more information.
+
+== Prerequisites
+Before following this how-to guide, ensure you have met the following prerequisites:
+
+* A CircleCI account, xref:first-steps#[you can sign up for free].
+* Familiarity with CircleCI configuration files. See the xref:introduction-to-yaml-configurations#[Introduction to YAML Configurations] guide for more information.
+* Familiarity with the concepts of URL and registry orbs, and the URL orb allow-list. #INSERT LINKS WHEN AVAILABLE#
+* xref:github-apps-integration#[GitHub App integration]. This is optional but recommended to get the best experience with centralized configs due to the ability to define pipelines in which the configuration files is stored outside the project repository.
+
+== Simple config override example
+
+This basic how-to guide demonstrates how to override a job in your workflow with a job from a URL orb. The orb is stored in a location that is accessible as defined by your org's allow-list.
+
+=== 1. Define your base configuration
+
+Create a configuration with standard jobs, to build and test an application:
+
+[source,yaml]
+----
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: cimg/base:2023.03
+    steps:
+      - checkout
+      - run: echo "Building application"
+  test:
+    docker:
+      - image: cimg/base:2023.03
+    steps:
+      - checkout
+      - run: echo "Running standard tests"
+
+workflows:
+  main:
+    jobs:
+      - build
+      - test:
+          requires:
+            - build
+----
+
+=== 2. Define the override job in your orb
+
+Create a URL orb for your custom test job in a location that is accessible for your project. In the next step you will ensure that the URL is included in your org's allow list.
+
+A URL orb can be very simple, just a YAML file that defines the job you want to use to override one of your standard jobs.
+
+In your orb, define the custom-test job that will replace the standard test job you defined in your base configuration:
+
+[source,yaml]
+----
+# my-orb
+jobs:
+  custom-test:
+    docker:
+      - image: cimg/node:18.0
+    steps:
+      - checkout
+      - run: echo "Running custom test suite"
+      - run: npm test
+----
+
+Notice that this file:
+
+* Does NOT include `version: 2.1`.
```

Comment:
It needs to include `version` to be a valid config file, just like published orbs.